libs/core/algorithms/CMakeLists.txt

```diff
@@ -17,6 +17,7 @@ set(algorithms_headers
     hpx/parallel/algorithms/copy.hpp
     hpx/parallel/algorithms/count.hpp
     hpx/parallel/algorithms/destroy.hpp
+    hpx/parallel/algorithms/detail/adjacent_difference.hpp
```

Comment:
`hpx/parallel/algorithms/detail/adjacent_difference.hpp` belongs into `<numeric>`, not `<algorithm>`

Reply:
Duh, this is the CMakeLists.txt, not the #include. Everything is fine! Sorry for the noise.